docs/data/data-grid/filtering-recipes/QuickFilterOutsideOfGrid.js

```diff
@@ -8,10 +8,9 @@ import { useDemoData } from '@mui/x-data-grid-generator';
 function MyCustomToolbar(props) {
   return (
     <React.Fragment>
-      {ReactDOM.createPortal(
-        <GridToolbarQuickFilter />,
-        document.getElementById('filter-panel'),
-      )}
+      <Portal container={() => document.getElementById('filter-panel')}>
```

Comment:
Could we directly use the selector here?
```suggestion
      <Portal container={document.getElementById('filter-panel')}>
```

As per docs:
![image](https://github.com/mui/mui-x/assets/12609561/960d38e6-2fdf-4a83-8936-879fab475175)

Reply:
It would break during server side rendering, I guess.